src/lib/django-activities/activities/notifiers/oauth/base.py

```diff
@@ -74,7 +74,7 @@ def get_credentials(cls, filename):
         The return value is a dictionary to initialize a
         requests_oauthlib.OAuth1
         """
-        if not os.path.exists(filename):
+        if not os.path.exists(filename) and not settings.TESTING:
```

Comment:
I said initialization code but if you can specify such code in a particular code block, that is much better.